lib/string_reverse.rb

```diff
@@ -18,3 +18,5 @@ def string_reverse(my_string)
   return my_string

```

Comment:
Minor: Since this method reverses the string in-place, you don't need to return my_string on lines 4 or 18 and these lines could simply be `return`.